### ISSUE TITLE
Unnecessary option name for track details

### ISSUE THREAD

**Issue by WillSmithTE** [WillSmithTE/mirlo#60]
![Image](https://github.com/user-attachments/assets/167c91b7-abd7-42cb-b255-63fde244c730)
Here is a pattern: on the left, the name of the option, and on the right, the option itself which essentially says the same thing with other words or a different syntax.
Maybe this is important for accessibility ?
If not, the title of the options on the left should be removed for clarity (you don't need to read the same information twice). If it is important for accessibility, then choose a single word or a couple of words to describe these options. For example, from top to bottom i'd choose: "Listed artists" (no change here), "Individual sale" and "Hype Circle promotion" (or simply "Promotion").

**Comment by WillSmithTE** [comment]
**@timglorioso** commented on 2025-06-18T02:14:06Z:

---

1. It's true that checkbox and radio button labels can be self-sufficient. In this case, Promotion benefits from some details.\n2. For Individual Sale, removing the left part may disrupt the visual flow of information and make the setting easier to miss. Just speculation.\n3. Redundant information isn't particularly helpful regardless of whether someone is using their eyes or a screen reader.\n4. ~It looks like neither label is associated with the checkbox (`for` requires `id`, not `name`), so that's something to address.~ I was mistaken\n5. I would consider adjusting the entire track editing layout so that labels are above their inputs, like in the small screen layout.

**Comment by WillSmithTE** [comment]
**@Cwpute** commented on 2025-06-18T11:24:54Z:

---

What you suggest in point 5 would indeed be a good starting point.\nRegarding "Promotion", i agree as well, "Hype Circle Promotion" would be a good name for that radio button, with more information added below for people discovering this feature.\nYour other points seem valid as well 👍

**Comment by WillSmithTE** [comment]
**@timglorioso** commented on 2025-09-18T21:39:24Z:

---

I've started working on this one. Converting the table to a list to improve semantic use of HTML.
